tensorbay/opendataset/UrbanObjectDetection/loader.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name, missing-module-docstring
+
+"""Dataloader of UrbanObjectDetection dataset."""
+
+import os
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import LabeledBox2D
+
+try:
+    import xmltodict
+except ModuleNotFoundError:
+    from tensorbay.opendataset._utility.mocker import xmltodict  # pylint:disable=ungrouped-imports
+
+_SEGMENT_NAMES = ("train", "val", "test", "trainval")
```

Comment:
Is `trainval` equal to `train` plus `val`?  If so, there is no need to add the `trainval` segment.

Reply:
trainval is equal to train plus val, but trainval is listed under ImageSets. Should I also remove the segment.